packages/react-reconciler/src/ReactFiberReconciler.old.js

```diff
@@ -265,6 +267,15 @@ export function updateContainer(
   const suspenseConfig = requestCurrentSuspenseConfig();
   const lane = requestUpdateLane(current, suspenseConfig);
 
+  if (enableSchedulingProfiling) {
+    let componentName = 'Unknown';
+    if (ReactCurrentFiberCurrent !== null) {
+      componentName =
+        getComponentName(ReactCurrentFiberCurrent.type) || componentName;
```

Comment:
Since `ReactCurrentFiberCurrent` is DEV-only, I think we should just remove the concept of component name from render-scheduled marks. It seems undesirable that "Unknown" could mean:
1. A component without a name is rendering.
2. No component is rendering.
3. A component is rendering, but we're in production mode so we don't know its name.

It's also odd that we're passing in the *name* of the current Fiber along with the root Fiber. Those always match up in other places.